examples/01_template/package.json

```
@@ -17,6 +17,8 @@
   "devDependencies": {
     "@types/react": "18.2.46",
     "@types/react-dom": "18.2.18",
+    "autoprefixer": "^10.4.16",
+    "tailwindcss": "^3.4.1",
```

We pin everything just in case.
```suggestion
    "autoprefixer": "10.4.16",
    "tailwindcss": "3.4.1",
```